write_column.go

```diff
@@ -14,20 +15,13 @@ type ColumnarStatement interface {
 }
 
 func (stmt *stmt) ColumnWriter() ColumnWriter {
-	stmt.ch.data.reserveColumns()
-	return stmt.ch.data
+	stmt.ch.block.Reserve()
+	return stmt.ch.block
 }
 
 func (stmt *stmt) ColumnWriterEnd(rows uint64) error {
-	writtenBlocks := stmt.counter / stmt.ch.blockSize
-	stmt.counter += int(rows)
-	stmt.ch.data.numRows += rows
-
-	var err error
-	if stmt.counter/stmt.ch.blockSize > writtenBlocks {
-		err = stmt.ch.data.write(stmt.ch.serverRevision, stmt.ch.conn)
-	}
-	return err
```

Comment:
Yes, that makes sense, but I would like to make the second interface for the driver (without support database/sql) to provide this functionality. I think, will be better to create new issue for it )